<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>850</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BANCO DE HORA</string>
  </property>
  <property name="windowIcon">
   <iconset resource="_imagens/imagens.qrc">
    <normaloff>:/newPrefix/clinica-_2_.ico</normaloff>:/newPrefix/clinica-_2_.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1200</width>
     <height>800</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1300</width>
     <height>850</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>1200</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1300</width>
        <height>80</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(202,0,88);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="nome_func">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(145, 145, 145);
font: 75 12pt &quot;Arial&quot;;
padding-left: 5px;</string>
         </property>
         <property name="text">
          <string>Nome do Funcionário</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="hora_func">
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(145, 145, 145);
font: 75 12pt &quot;Arial&quot;;
padding-left: 5px;</string>
         </property>
         <property name="text">
          <string>Horário Registrado</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1100</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1200</width>
        <height>600</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="imgLabel">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>0</y>
            <width>900</width>
            <height>550</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>700</width>
            <height>500</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>900</width>
            <height>700</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
border:1px solid;
border-color: rgb(212, 212, 212);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QFrame" name="info">
          <property name="geometry">
           <rect>
            <x>450</x>
            <y>180</y>
            <width>321</width>
            <height>201</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame#info{
background-color: rgb(202,0,88);
border: 1px solid white;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="info_text">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>50</y>
             <width>281</width>
             <height>101</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(202,0,88);
color: white;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </widget>
         <widget class="QFrame" name="pega_nome">
          <property name="geometry">
           <rect>
            <x>450</x>
            <y>180</y>
            <width>321</width>
            <height>201</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame#pega_nome{
background-color: rgb(202,0,88);
border: 1px solid white;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="info_text_2">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>10</y>
             <width>231</width>
             <height>61</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(202,0,88);
color: white;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;INSIRA O NOME DO &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;FUNCIONARIO&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QLineEdit" name="nome">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>90</y>
             <width>301</width>
             <height>41</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="enviar_nome">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>150</y>
             <width>121</width>
             <height>28</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#enviar_nome{
color: rgb(202,0,88);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border: 2px solid  rgb(202,0,88);
border-radius: 5px;
}
QPushButton#enviar_nome:hover:pressed{
background-color: rgb(202,0,88);
color: white;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border: 2px solid white;
border-radius: 5px;
}</string>
           </property>
           <property name="text">
            <string>CRIAR QR-CODE</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="minimumSize">
       <size>
        <width>1200</width>
        <height>150</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1300</width>
        <height>150</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(202,0,88);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="sair">
          <property name="geometry">
           <rect>
            <x>370</x>
            <y>10</y>
            <width>100</width>
            <height>100</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#sair{
background-image: url(:/newPrefix/sair_branco.png);
background-repeat:no-repeat;
background-position:center;
border: 1px solid white;
border-radius: 50px;}
QPushButton#sair:hover:pressed{
background-color: rgb(244,123,166);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="abrir_camera">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>10</y>
            <width>100</width>
            <height>100</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#abrir_camera{
background-image: url(:/newPrefix/camera_branca.png);
background-repeat:no-repeat;
background-position:center;
border: 1px solid white;
border-radius: 50px;}
QPushButton#abrir_camera:hover:pressed{
background-color: rgb(244,123,166);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="fechar_camera">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>10</y>
            <width>100</width>
            <height>100</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#fechar_camera{
background-image: url(:/newPrefix/desligar_camera_branca.png);
background-repeat:no-repeat;
background-position:center;
border: 1px solid white;
border-radius: 50px;}
QPushButton#fechar_camera:hover:pressed{
background-color: rgb(244,123,166);

}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="qr_code">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>10</y>
            <width>100</width>
            <height>100</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="whatsThis">
           <string>Cria Qr-Code</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#qr_code{
background-image: url(:/newPrefix/criar_qr_code.png);
background-repeat:no-repeat;
background-position:center;
border: 1px solid white;
border-radius: 50px;}
QPushButton#qr_code:hover:pressed{
background-color: rgb(244,123,166);

}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="_imagens/imagens.qrc"/>
 </resources>
 <connections/>
</ui>
